tools/sgen/input/FluentUIStyle.yml

```diff
@@ -403,6 +403,57 @@ AP_MSFPersonaViewTokens:
   labelFont: $Typography.headline
   sublabelColor: $Colors.Foreground.neutral1
 
+AP_MSFPillButtonBarTokens:
+  cornerRadius: $Spacing.medium
+  maxButtonsSpacing: 10pt
+  minButtonsSpacing: $Spacing.xSmall
+  minButtonVisibleWidth: $Spacing.large
+  minButtonWidth: 56pt
+  minHeight: 28pt
+  sideInset: $Spacing.medium
+  hoverBackgroundColor: [ primary: "FluentUIColor(light: NamedColor(grey94), dark: NamedColor(grey8))",
+                          onBrand: "FluentUIColor(light: $Colors.Brand.shade10, dark: NamedColor(grey8))" ]
+
+AP_MSFPillButtonTokens:
+  bottomInset: 6pt
+  horizontalInset: $Spacing.medium
+  topInset: 6pt
+  unreadDotOffsetX: 6pt
+  unreadDotOffsetY: 3pt
+  unreadDotSize: 6pt
+  backgroundColor: [ rest: "FluentUIColor(light: NamedColor(grey94), dark: NamedColor(grey8))",
```

Comment:
I know this is a spec issue, but it's still frustrating that so many of these colors are `NamedColor` values (e.g. Global Tokens).

Reply:
I agree, but I don't have much choice in the matter right now :(